src/server/commands/Command.ts

```diff
@@ -11,7 +11,7 @@ abstract class Command {
             (source: number, args: string[], rawCommand: string) => {
                 this.execute(source.toString(), args, rawCommand);
             },
-            false
+            true
```

Comment:
The command cannot be restricted because vRP it isn't compatible with the FiveM permissions.